docs/src/kernel/advanced-instructions.md

```diff
@@ -58,18 +62,32 @@ Then, in the interactive shell, run the following script to start the syscall te
 
 ### Using GDB to Debug
 
-To debug Asterinas using [QEMU GDB remote debugging](https://qemu-project.gitlab.io/qemu/system/gdb.html), one could compile Asterinas in the debug profile, start an Asterinas instance and run the GDB interactive shell in another terminal.
+To debug Asterinas using [QEMU GDB remote debugging](https://qemu-project.gitlab.io/qemu/system/gdb.html),
+one could compile Asterinas in the debug profile,
+start an Asterinas instance and run the GDB interactive shell in another terminal.
 
-First, start a GDB-enabled VM of Asterinas and wait for debugging connection:
+Start a GDB-enabled VM of Asterinas with OSDK and wait for debugging connection:
 
 ```bash
-make run GDB_SERVER=1 ENABLE_KVM=0
+/asterinas/kernel# cargo osdk debug server
 ```
 
-Next, start a GDB client in another terminal.
+Then debug via [a GDB client in terminal](#terminal), or [VS Code](#vs-code).
+
+#### Terminal
```

Comment:
Um, a little bit weird using headings here for me.

Reply:
Now in list instead.